src/DynamoCore/Graph/Workspaces/CustomNodeWorkspaceModel.cs

```diff
@@ -235,7 +235,7 @@ public bool IsVisibleInDynamoLibrary
 
         private bool isVisibleInDynamoLibrary;
 
-        protected override void RequestRun()
+        internal override void RequestRun()
```

Comment:
It was upgraded to `internal` in the base class as well, as it's called from an extension method now.